content/en/docs/contributing/release-process.md

```diff
@@ -27,35 +36,47 @@ The process for cutting a minor release is as follows:
 
 1. Ensure upgrading document exists.
 
-2. Ensure all strings of versions have been updated.
+2. Create a new release branch (e.g. `release-0.5`)
 
-- in the [`jetstack/cert-manager`](https://github.com/jetstack/cert-manager)
-  repository
- - `deploy/charts/cert-manager/README.md`
-- in the [`cert-manager/webisite`](https://github.com/cert-manager/website)
-  repository
- - `content/en/docs/installation/compatibility.md`
- - `content/en/docs/installation/kubernetes.md`
- - `content/en/docs/installation/openshift.md`
+3. Push it to the `jetstack/cert-manager` repository
 
-3. Create a new release branch (e.g. `release-0.5`)
+4. Run `cmrel stage`
 
-4. Push it to the `jetstack/cert-manager` repository
+To stage a release of the 'release-1.0' branch to the default staging bucket,
+overriding the release version as 'v1.0.0':
 
-5. Gather release notes since the previous release:
+```#bash
+ cmrel stage --git-ref=release-1.0 --release-version=v1.0.0
+```
 
-Download, install and run the latest version of release-notes:
+4. [Create release notes](#release-notes)
+
+5. Run `cmrel publish`
+
+5.1. First do a dry-run, to ensure that all the staged resources are valid.
 
-```bash
-$ go get k8s.io/release; go install $GOPATH/src/k8s.io/release/cmd/release-notes/.
-$ mkdir -p design/release-notes/release-*X.Y*
-$ export GITHUB_TOKEN=*your-token*
-$ $GOPATH/bin/release-notes -release-version v*X.Y* -github-repo cert-manager -github-org jetstack -requiredAuthor "" -start-sha=$(git rev-parse *X.Y-1.0*) -end-sha=$(git rev-parse HEAD) -output design/release-notes/release-*X.Y*/draft-release-notes.md
 ```
-Add additional blurb, notable items and characterize change log.
+cmrel publish --release-name <RELEASE_NAME>
+```
+Where `<RELEASE_NAME>` is the unique build ID printed by the earlier `cmrel stage` command.
 
+5.2 Next publish the release artifacts for real
 
-Finally, create a new tag taken from the release branch, e.g.`v0.5.0`.
+If the last step succeeded, you can now re-run the `cmrel publish` with the `--nomock` argument to actually publish the release articacts to Github, quay.io, helm hub etc.
+
+```
+cmrel publish --nomock --release-name <RELEASE_NAME>
+```
+
+NOTE: At this stage there will be a draft release on Github and a live release on HelmHub.
+So you must now complete the release process quickly otherwise users of the latest release on HelmHub will encounter errors.
```

Comment:
Let's be a bit more specific here for drive by readers who think the world is on fire: it causes the manual CRD install URL not to be available yet